File: Linkpoint/src/main/java/com/linkpoint/ui/common/OnInterceptTouchEventListener.java
package com.linkpoint.ui.common;
import java.util.*;

import android.view.MotionEvent;

public interface OnInterceptTouchEventListener {
    boolean dispatchTouchEvent(MotionEvent motionEvent);

    boolean onInterceptTouchEvent(MotionEvent motionEvent);

    boolean onTouchEvent(MotionEvent motionEvent);
}

File: Linkpoint/src/main/java/com/linkpoint/ui/common/OnInterceptTouchEventListener.kt
package com.linkpoint.ui.common

import android.view.MotionEvent

interface OnInterceptTouchEventListener {
    fun dispatchTouchEvent(motionEvent: MotionEvent): Boolean
    fun onInterceptTouchEvent(motionEvent: MotionEvent): Boolean
    fun onTouchEvent(motionEvent: MotionEvent): Boolean
}